lib/registries/addon/components/editable-field/template.hbs

```diff
@@ -0,0 +1,20 @@
+{{#if @registration.userHasAdminPermission}}
+    <div local-class='Editable'>
+        <h4>{{t @title}}</h4>
```

Comment:
```suggestion
        <h4>{{@title}}</h4>
```
Better not to assume it's a translation key, let the invoker take care of that. What if the translation has parameters?

Reply:
Good call.